posthog/api/test/dashboards/__snapshots__/test_dashboard.ambr

```diff
@@ -268,6 +271,7 @@
          "posthog_team"."ingested_event",
          "posthog_team"."session_recording_opt_in",
          "posthog_team"."capture_console_log_opt_in",
+         "posthog_team"."capture_performance_opt_in",
```

Comment:
I'm not sure why this snapshot changed, but it isn't to do with this change so 🤷

Reply:
I'm guessing the pr that added the migration didn't run these snapshots 🤔